### ISSUE TITLE
Server versions that don't exist?

### ISSUE THREAD

**Issue by JustinWebDev** [MicrosoftDocs/windowsserverdocs#5373]
Only Server 2016 and 2019 can be purchased or downloaded for Evaluation.  I have not found a way to get any of the other versions of Server.  Thus, it is very confusing for MS to document releases of software that are apparently unattainable.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: fbf74885-9525-9828-95c9-0ab9167b1f35
* Version Independent ID: f0a39382-120b-0cfa-692f-621124a23f37
* Content: [Windows Server release information](https://docs.microsoft.com/en-us/windows-server/get-started/windows-server-release-info)
* Content Source: [WindowsServerDocs/get-started/windows-server-release-info.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/get-started/windows-server-release-info.md)
* Product: **windows-server**
* Technology: **getting-started**
* GitHub Login: @JasonGerend
* Microsoft Alias: **jgerend**

**Comment by illfated** [comment]
https://docs.microsoft.com/en-us/windows-server/get-started-19/servicing-channels-19#semi-annual-channel

[ … ]

The Semi-Annual Channel is available to volume-licensed customers with Software Assurance (https://www.microsoft.com/licensing/licensing-programs/software-assurance-default.aspx), as well as via the Azure Marketplace or other cloud/hosting service providers and loyalty programs such as Visual Studio Subscriptions.

**Comment by illfated** [comment]
If you need to test Windows Server on a Trial License (180 days) before buying, you can only download these versions:

| Windows Server products           | Trial period |
| :-------------------------------- | :----------- |
| Windows Server 2019               | 180 days     |
| Windows Server 2019 Essentials    | 180 days     |
| Hyper-V Server 2019               | Unlimited    |
| Windows Admin Center              | Free         |
| Windows Server 2016               | 180 days     |
| Hyper-V Server 2016               | Unlimited    |
| Windows Server 2016 Essentials    | 180 days     |
| Windows Server 2012 R2            | 180 days     |
| Windows Server 2012 R2 Essentials | 180 days     |
| Hyper-V Server 2012 R2            | Unlimited    |
| Windows Server 2012               | 180 days     |
| Hyper-V Server 2012               | Unlimited    |

**Comment by illfated** [comment]
The Semi-Annual Channel versions are targeted to existing Microsoft customers who need a more rapid Server OS version progression than the average user, either due to high upgrade frequency on the other software solutions or to be on "the cutting edge" of their software in general.

**Comment by JustinWebDev** [comment]
> If you need to test Windows Server on a Trial License (180 days) before buying, you can only download these versions:
> 
> Windows Server products	Trial period
> Windows Server 2019	180 days
[...]
> Hyper-V Server 2012	Unlimited

Yes, I can download those.  I already stated that.

The point is I can't even purchase anything except what you noted.  I've looked at several retailers and they do not offer the odd versions.  I am not aware of Azure Marketplace, etc.  Maybe "Semi-Annual Channel" is the key.  It is like a secret - only the elite can even find it!

**Comment by illfated** [comment]
> "The Semi-Annual Channel is available to volume-licensed customers with Software Assurance".

Fair enough. Looks like the likely place to get it would be to get a Software Assurance subscription first (no idea about those prices, probably ranks up there with volume licenses, for all I know.

**Comment by IngridAtMicrosoft** [comment]
#assign:IngridAtMicrosoft
